src/distribution/lognormal.rs

```diff
@@ -0,0 +1,223 @@
+use distribution;
+use source::Source;
+
+/// A lognormal distribution.
+#[derive(Clone, Copy)]
+pub struct Lognormal {
+    mu: f64,
+    sigma: f64,
+}
+
+impl Lognormal {
+    /// Create a lognormal distribution with location `mu` and scale `sigma`.
+    ///
+    /// It should hold that `sigma > 0`.
+    #[inline]
+    pub fn new(mu: f64, sigma: f64) -> Self {
+        should!(sigma > 0.0);
+        Lognormal { mu: mu, sigma: sigma }
```

Comment:
I thought about it. No any objection. Also I can take `mu` and `sigma` from gaussian holder field and don't duplicate it. Should I have to do it, or we have to keep parameters separately?